### PR TITLE
Added environments

### PR DESCRIPTION
**Pull request by jonvanputten**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Do some configuration
 config :cached_contentful,
     space_id: "YOUR_SPACE_ID",
     access_token: "YOUR_ACCESS_TOKEN",
+    environment: "YOUR_ENVIRONMENT",
```
Comment (EvertVerboven) [inline README.md:29]: Does this mean we have to set env variables in the container of are they used from the config.exs files? The latter would have my preference.

Comment (EvertVerboven) [reply README.md:29]: Or is this dependant on the configuration in the "parent" application?

**Review by EvertVerboven — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jonvanputten) [reply README.md:29]: This is all from the config file and needs to be set in the child application

Comment (EvertVerboven) [reply README.md:29]: Ok so in the case of villa-augustus, this config variable wil be set in the config files of that application?

**Pull request merged**